static/js/components/tiles/answer_message_tile.tsx

```diff
@@ -40,43 +41,88 @@ export interface AnswerMessageTilePropType {
   displayValue?: DisplayValueSpec;
 }
 
+interface AnswerMessageTileData {
+  // Inline value to show as the answer
+  displayValue: DisplayValueSpec;
+  // Name of the entity to show the answer for
+  entityName: string;
+}
+
+/**
+ * Bold and italicize an entity's name in the title
+ * @param title tile title containing entity's name
+ * @param entityName name of the entity to highlight
+ * @returns JSX element containing stylized element
+ */
+function highlightEntities(title: string, entityName: string): JSX.Element {
+  const entityIndex = title.indexOf(entityName);
+  if (entityIndex == -1) {
+    return <span>{title}</span>;
+  }
+  return (
+    <span>
+      {title.slice(0, entityIndex)}
+      <b>
+        <i>{entityName}</i>
+      </b>
+      {title.slice(entityIndex + entityName.length, title.length)}
```

Comment:
nit: for the slice method, you can leave the second argument empty to return the rest of the string, so this is equivalent to `title.slice(entityIndex + entityName.length)`

Reply:
Thanks! did not know this. Simplified to not include the second argument.